src/helpers/spatialHelper.ts

```diff
@@ -21,3 +22,56 @@ export const englandAndWales: GeographyInfo = {
     ],
   },
 };
+
+// uk bounding box, taken from UK bbox value used in API
+// https://github.com/ONSdigital/dp-geodata-api/blob/develop/pkg/geodata/coords.go
+export const UKBbox: Bbox = {
+  east: 1.76,
+  north: 58.64,
+  west: -7.57,
+  south: 49.92,
+};
+
+/*
+  Snap map bounding box map tile grid to make bouding box requests cachable.
+*/
+export const snapBboxToMapGrid = (bbox: Bbox, zoom: number) => {
+  console.debug("Snapping bbox:");
+  console.debug(bbox);
+
+  // round down zoom level to ensure snap is to integer tile grid
+  const floorZoom = Math.floor(zoom);
+
+  // if zoom is lower than minZoomToSnapBBoxes, just return bbox for all of UK
+  const minZoomToSnapBBoxes = 6;
+  if (floorZoom <= minZoomToSnapBBoxes) {
+    console.debug(`Rounded-down zoom level ${floorZoom} <= min of ${minZoomToSnapBBoxes} - using uk bbox:`);
+    console.debug(UKBbox);
+    return UKBbox;
+  }
+
+  // get map tiles for current zoom that contain bbox corners
+  const NETile = pointToTile(bbox.east, bbox.north, floorZoom);
+  const SWTile = pointToTile(bbox.west, bbox.south, floorZoom);
```

Comment:
"However, you should capitalize acronyms that consist of only two characters, such as System.IO instead of System.Io." - https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN
Recognise its a consistency thing though. Will change.

Reply:
done